ext/native-decls/GetVehicleAlarmTimeLeft.md

```diff
@@ -7,9 +7,11 @@ apiset: client
 ```c
 int GET_VEHICLE_ALARM_TIME_LEFT(Vehicle vehicle);
 ```
-
+Returns vehicle alarm time left(Millisecond).<br />
```

Comment:
Why the `<br />`? Similarly, weird phrasing - is English not your native language?

Reply:
I prefer to use this to break lines but if it is unusual for you I will use double space, no is not my native language.